.github/docs/GIT_WORKFLOW.md

```
@@ -0,0 +1,257 @@
+# Git Workflow Guide
+
+### 목차
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)